resume.md

```diff
@@ -1,49 +1,51 @@
-# Alex Takahashi
+# RAHIL SHAH
 
-(123) 456 - 7890 | email@gmail.com
+(647) 972 - 0150 | rahil190@gmail.com
+
+rahil.me | linkedin.com/in/rahilllshah
 
-alextakahashi.github.io | github.com/alextakahashi | inkedin.com/in/alextakahashi/
 
 ## EDUCATION
-**University of California Berkeley** | B.S. Bioengineering, Computer Science Minor ... *May 2017*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**McMaster University** | BS Computer Science ... *Jun '22*
+
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Design & Prototyping**: Adobe CC Software, Figma,  InVision, Sketch, Visual design, interaction design, responsive design, prototyping, user-centered design, usability testing
+* **Development**: HTML, CSS, JavaScript, Java, C, Python, SQL, Git
```

Comment:
Your Programming languages should be first and foremost assuming youre applying for SWE.